oauth2/fosite_store_sql.go

```diff
@@ -298,9 +301,9 @@ func (s *FositeSQLStore) deleteSession(ctx context.Context, signature string, ta
 	return nil
 }
 
-func (s *FositeSQLStore) CreateSchemas() (int, error) {
+func (s *FositeSQLStore) CreateSchemas(driver string) (int, error) {
```

Comment:
Maybe rename this to `dbName`, otherwise this would be considered the driver which would be expected to be `postgres` for CockroachDB.